ts/client/src/accounts/mangoAccount.ts

```diff
@@ -182,6 +182,17 @@ export class MangoAccount {
     return this.frozenUntil.lt(new BN(Date.now() / 1000));
   }
 
+  public async tokenPositionsForNotConfidentOrStaleOracles(
```

Comment:
So I am thinking like this, the UI basically calls this, and knows which banks have deposits, and which have borrows, 

for borrows - it asks user to close borrow
for deposits - it asks user to withdraw all using the new helper in client.ts